includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -369,7 +436,7 @@ public function get_normalized_postal_code( $postcode, $country ) {
 	 *
 	 * @return  void
 	 */
-	public function add_order_meta( $order_id, $posted_data ) {
+	public function add_order_meta( $order_id, $posted_data = null ) {
```

Comment:
Can we delete this `$posted_data`?

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/4264/commits/4ccb5ccfaaa505de2aa0eac8dccaa8e0fe2fa312